docs/guides/custom-functions.md

```diff
@@ -233,6 +233,18 @@ module.exports = (targetVal, _opts, paths) => {
 };
 ```
 
+Furthermore, if your function performs any fs calls to obtain dictionaries and similar, you may want to leverage cache.
```

Comment:
@P0lip The doc is pretty clear to me. However, from a user standpoint, there's no information with regards to _how_ I can leverage the cache. 

Could you please add a simple sample?

Reply:
Ah yeah, forgot about an example.